packages/opds-web-client/src/components/CatalogLink.tsx

```diff
@@ -12,20 +12,20 @@ export interface CatalogLinkProps extends React.HTMLProps<{}> {
 export default class CatalogLink extends React.Component<CatalogLinkProps, {}> {
   context: NavigateContext;
 
-  static contextTypes: React.ValidationMap<NavigateContext> = {
```

Comment:
What happened to the `NavigateContext` type?

Reply:
See my reply in the main thread @EdwinGuzman